spec/support/fake_heroku.rb

```diff
@@ -6,8 +6,9 @@ def initialize(args)
   end
 
   def run!
-    if @args.first == "plugins"
-      puts "heroku-pipelines@0.29.0"
+    if @args.first == "help"
+      # spaces in front are intentional
+      puts "  pipelines      #  manage collections of apps in pipelines"
```

Comment:
I agree it's better not to need comments. Also, I think this feature will be more reliable if we only assert on the presence of `pipelines`, without any information on specific formatting. That way, if in a future the amount of spaces changes, behavior will always be the same.

Can we make it so that the spaces are not relevant?

Thank you for your contribution!